extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -333,20 +358,41 @@ class AmpAccordion extends AMP.BaseElement {
                   sectionIter.removeAttribute('expanded');
                 }
                 sectionIter.children[0].setAttribute('aria-expanded', 'false');
+                this.setRenderSubtreeIfEnabled(
+                  sectionIter.children[1],
+                  'invisible skip-viewport-activation'
+                );
               }
             });
           }
         } else {
           this.triggerEvent_('collapse', section, trust);
           section.removeAttribute('expanded');
           header.setAttribute('aria-expanded', 'false');
+          this.setRenderSubtreeIfEnabled(
+            content,
+            'invisible skip-viewport-activation'
+          );
         }
       }, section);
     }
     this.currentState_[contentId] = !isSectionClosedAfterClick;
     this.setSessionState_();
   }
 
+  /**
+   * If Display Locking API is enabled, set the renderSubtree attribute
+   * on the given element with the given value.
+   * @param {Element} element
+   * @param {string} value
+   */
+  setRenderSubtreeIfEnabled(element, value) {
+    if (!isDisplayLockingEnabledForAccordion(this.win) || !element) {
+      return;
+    }
+    element.renderSubtree = value;
```

Comment:
I'm pretty sure you will need to add this to the Closure Compiler's browser externs to prevent renaming during compilation. Should either use the attribute (via `setAttribute()`) or use bracket access for now.

Reply:
For now we should just add it to our own Closure extern: it still changes a lot so pushing it to Closure's shared externs is not going to fly. But I'd prefer the property access here since we also use the existence of the property as a enabling condition.

Reply:
Actually I take it back a little. We will soon switch to the newly-minted CSS properties for this. So property/attribute is not important.

Reply:
Actually I take it back a little. We will soon switch to the newly-minted CSS properties for this. So property/attribute is not important.